commit 9b0ba7594421f916f33b1bd8640295be1b91177e
Author: John Arthorne <johna>
Date:   Wed Apr 29 16:05:36 2009 +0000

    Bug 274319 Remove unused strings

0	2	bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/Messages.java
0	1	bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/messages.properties
0	8	bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/Messages.java
0	5	bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/messages.properties
1	3	bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/Messages.java
0	2	bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/messages.properties
0	2	bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/Messages.java
0	1	bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/messages.properties
1	3	bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/Messages.java
0	2	bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/messages.properties
0	7	bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/Messages.java
0	6	bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/messages.properties
0	8	bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/Messages.java
0	5	bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/messages.properties
0	2	bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/Messages.java
0	2	bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/messages.properties
1	6	bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/Messages.java
1	6	bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/messages.properties
0	2	bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java
0	2	bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties
0	1	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
0	1	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
0	17	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
0	18	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
1	8	bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/Messages.java
1	6	bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/messages.properties
diff --git a/bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/Messages.java b/bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/Messages.java
index 74224a5..f0d1fa8 100644
--- a/bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/Messages.java
@@ -19,8 +19,6 @@ public class Messages extends NLS {
 
 	public static String _0_is_not_a_valid_qualifier_in_osgi_1;
 
-	public static String _0_is_not_an_osgi_version;
-
 	public static String array_can_not_be_empty;
 
 	public static String array_can_not_have_character_group;
diff --git a/bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/messages.properties b/bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/messages.properties
index dd2f51b..9f46d1e 100644
--- a/bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/internal/provisional/p2/core/messages.properties
@@ -11,7 +11,6 @@
 _0_is_not_a_positive_integer_in_osgi_1=The {0} value is not a positive in OSGi version: "{1}"
 _0_is_not_a_string_in_osgi_1=The {0} value is not a string in OSGi version: "{1}"
 _0_is_not_a_valid_qualifier_in_osgi_1=The {0} value is invalid in OSGi version: "{1}"array_can_not_be_empty=Array format can not be empty
-_0_is_not_an_osgi_version=The string "{0}" does not represent an OSGi version
 array_can_not_be_empty=Array format can not be empty
 array_can_not_have_character_group=Array format can not have a character group
 auto_can_not_have_pad_value=Auto format can not have a pad value
diff --git a/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/Messages.java b/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/Messages.java
index f888319..9468e23 100644
--- a/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/Messages.java
@@ -24,8 +24,6 @@ class Messages extends NLS {
 
 	public static String error_persisting_profile;
 
-	public static String failed_creating_metadata_cache;
-
 	public static String ParameterizedProvisioningAction_action_or_parameters_null;
 
 	public static String profile_does_not_exist;
@@ -38,13 +36,7 @@ class Messages extends NLS {
 	public static String Profile_Null_Profile_Id;
 	public static String Profile_Parent_Not_Found;
 
-	public static String ProfilePreferences_save_failed;
-
 	public static String ProfilePreferences_saving;
-	public static String ProfilePreferences_load_failed;
-
-	public static String ProfilePreferences_nullDir;
-	public static String ProfilePreferences_Profile_not_found;
 
 	public static String reg_dir_not_available;
 
diff --git a/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/messages.properties b/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/messages.properties
index 5750a36..0195054 100644
--- a/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/p2/engine/messages.properties
@@ -17,7 +17,6 @@ download_no_repository=No artifact repository available.
 
 error_parsing_profile=Error parsing profile {0}.
 error_persisting_profile=Error persisting profile {0}.
-failed_creating_metadata_cache=Metadata cache could not be created.
 io_FailedRead=Failed to read profile repository at {0}
 io_NotFound=No repository found at {0}
 
@@ -49,8 +48,4 @@ thread_not_owner=Thread not lock owner.
 profile_lock_not_reentrant=Lock failed. Profile does not permit reentrant locking.
 shared_profile_not_found=Shared profile {0} not found.
 
-ProfilePreferences_save_failed=An error occurred while saving preferences for profile {0}
 ProfilePreferences_saving=Saving profile preferences
-ProfilePreferences_load_failed=An error occurred while loading preferences for profile {0}
-ProfilePreferences_nullDir=Profile data directory is unexpectedly null
-ProfilePreferences_Profile_not_found=Requested profile {0} not located for preferences.
\ No newline at end of file
diff --git a/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/Messages.java b/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/Messages.java
index a20a4e5..6f3c710 100644
--- a/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/Messages.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2007, 2008 IBM Corporation and others.
+ * Copyright (c) 2007, 2009 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
@@ -26,7 +26,6 @@ public class Messages extends NLS {
 	public static String null_phases;
 	public static String null_phaseset;
 	public static String null_profile;
-	public static String operand_error;
 	public static String phase_error;
 	public static String phase_not_started;
 	public static String phase_started;
@@ -40,7 +39,6 @@ public class Messages extends NLS {
 	public static String phase_undo_error;
 	public static String session_prepare_error;
 	public static String session_commit_error;
-	public static String session_rollback_error;
 	public static String null_operand;
 	public static String operand_started;
 	public static String operand_not_started;
diff --git a/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/messages.properties b/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/messages.properties
index 80f0867..383abc9 100644
--- a/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/internal/provisional/p2/engine/messages.properties
@@ -19,7 +19,6 @@ null_profile=Profile must not be null.
 touchpoint_prepare_error=An error occurred while preparing the engine operation for the {0} touchpoint.
 touchpoint_commit_error=An error occurred while committing the engine operation for the {0} touchpoint.
 touchpoint_rollback_error=An error occurred while rolling back the engine operation for the {0} touchpoint.
-operand_error=An error occurred processing operand: {0}. 
 phase_error=An error occurred during the {0} phase.
 phase_undo_error=An error occurred while undoing the {0} phase.
 phase_not_started=There is no phase to end.
@@ -28,7 +27,6 @@ phaseid_not_positive=Phase weight must be positive.
 phaseid_not_set=Phase id must be set.
 session_prepare_error=An error occurred while preparing the engine session for profile: {0}.
 session_commit_error=An error occurred while committing the engine session for profile: {0}.
-session_rollback_error=An error occurred while rolling back the engine session for profile: {0}.
 null_operand=Operand must not be null.
 operand_started=An operand is already started.
 operand_not_started=There is no operand to end.
diff --git a/bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/Messages.java b/bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/Messages.java
index 74224a5..f0d1fa8 100644
--- a/bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/Messages.java
@@ -19,8 +19,6 @@ public class Messages extends NLS {
 
 	public static String _0_is_not_a_valid_qualifier_in_osgi_1;
 
-	public static String _0_is_not_an_osgi_version;
-
 	public static String array_can_not_be_empty;
 
 	public static String array_can_not_have_character_group;
diff --git a/bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/messages.properties b/bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/messages.properties
index dd2f51b..9f46d1e 100644
--- a/bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.metadata/src/org/eclipse/equinox/internal/provisional/p2/metadata/messages.properties
@@ -11,7 +11,6 @@
 _0_is_not_a_positive_integer_in_osgi_1=The {0} value is not a positive in OSGi version: "{1}"
 _0_is_not_a_string_in_osgi_1=The {0} value is not a string in OSGi version: "{1}"
 _0_is_not_a_valid_qualifier_in_osgi_1=The {0} value is invalid in OSGi version: "{1}"array_can_not_be_empty=Array format can not be empty
-_0_is_not_an_osgi_version=The string "{0}" does not represent an OSGi version
 array_can_not_be_empty=Array format can not be empty
 array_can_not_have_character_group=Array format can not have a character group
 auto_can_not_have_pad_value=Auto format can not have a pad value
diff --git a/bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/Messages.java b/bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/Messages.java
index 7f9ba28..ed027a0 100644
--- a/bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/Messages.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2007, 2008 IBM Corporation and others. All rights reserved. This
+ * Copyright (c) 2007, 2009 IBM Corporation and others. All rights reserved. This
  * program and the accompanying materials are made available under the terms of
  * the Eclipse Public License v1.0 which accompanies this distribution, and is
  * available at http://www.eclipse.org/legal/epl-v10.html
@@ -15,8 +15,6 @@ public class Messages extends NLS {
 
 	public static String exception_errorConverting;
 	public static String exception_stateAddition;
-	public static String exception_artifactRepoNotWritable;
-	public static String exception_metadataRepoNotWritable;
 	public static String exception_errorReadingManifest;
 	public static String exception_errorLoadingManifest;
 	public static String exception_noPluginConverter;
diff --git a/bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/messages.properties b/bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/messages.properties
index 85e07de..caa6191 100644
--- a/bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.publisher/src/org/eclipse/equinox/internal/p2/publisher/messages.properties
@@ -10,8 +10,6 @@
 ###############################################################################
 exception_errorConverting = An error occurred while generating manifest for {0}.
 exception_stateAddition = An error has occurred while adding the bundle {0}.
-exception_artifactRepoNotWritable = Artifact repository is not writable: {0}.
-exception_metadataRepoNotWritable = Metadata repository not writable: {0}.
 exception_errorReadingManifest = An error occurred while parsing the bundle manifest {0}: {1}.
 exception_errorLoadingManifest = An error occurred while loading the manifest {0}.
 exception_noBundlesOrLocations=No bundles or locations provided.
diff --git a/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/Messages.java b/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/Messages.java
index 692ebde..3edd78f 100644
--- a/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/Messages.java
@@ -18,14 +18,8 @@ public class Messages extends NLS {
 	public static String differentEntry;
 	public static String missingEntry;
 	public static String ioexception;
-	public static String ioexceptioninentry;
-	public static String filenotfoundexception;
-
-	public static String disassembler_description;
 	public static String disassembler_opentypedeclaration;
 	public static String disassembler_closetypedeclaration;
-	public static String disassembler_parametername;
-	public static String disassembler_localvariablename;
 	public static String disassembler_endofmethodheader;
 	public static String disassembler_begincommentline;
 	public static String disassembler_fieldhasconstant;
@@ -33,7 +27,6 @@ public class Messages extends NLS {
 	public static String disassembler_sourceattributeheader;
 	public static String disassembler_enclosingmethodheader;
 	public static String disassembler_exceptiontableheader;
-	public static String disassembler_arraydimensions;
 	public static String disassembler_innerattributesheader;
 	public static String disassembler_inner_class_info_name;
 	public static String disassembler_outer_class_info_name;
diff --git a/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/messages.properties b/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/messages.properties
index a1b0812..e8c3956 100644
--- a/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/comparator/messages.properties
@@ -12,17 +12,12 @@ differentNumberOfEntries=Difference in [{0}]: {1} contains {2} files and {3} con
 differentEntry=Difference found for {0} within [{1}] from {2}
 missingEntry=Missing {0} within [{1}] from {2}
 ioexception=IOException comparing {0} and {1} 
-ioexceptioninentry=IOException checking {0} within {1}
-filenotfoundexception=FileNotFoundException checking {0} within {1}
 
 ### Disassembler messages
 
 ### disassembler
-disassembler_description = Default classfile disassembler
 disassembler_opentypedeclaration =\ '{'
 disassembler_closetypedeclaration = }
-disassembler_parametername = arg
-disassembler_localvariablename = local_{0}
 disassembler_endofmethodheader = ;
 disassembler_begincommentline = //\ 
 disassembler_fieldhasconstant =\ =\ 
@@ -30,7 +25,6 @@ disassembler_endoffieldheader = ;
 disassembler_sourceattributeheader = Compiled from\ 
 disassembler_enclosingmethodheader = Enclosing Method:
 disassembler_exceptiontableheader = Exception Table:
-disassembler_arraydimensions = []
 disassembler_innerattributesheader = Inner classes:
 disassembler_inner_class_info_name = inner class info:
 disassembler_outer_class_info_name = outer class info:
diff --git a/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/Messages.java b/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/Messages.java
index e708b52..43ba46a 100644
--- a/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/Messages.java
@@ -27,12 +27,9 @@ public class Messages extends NLS {
 	public static String AbstractRepositoryTask_unableToFind;
 
 	public static String CompositeRepository_composite_repository_exists;
-	public static String CompositeRepository_create_colocated_composite_repos;
-	public static String CompositeRepository_creating_composite_repositories;
 	public static String CompositeRepository_default_artifactRepo_name;
 	public static String CompositeRepository_default_metadataRepo_name;
 
-	public static String no_valid_IUs;
 	public static String no_artifactRepo_manager;
 	public static String no_metadataRepo_manager;
 	public static String no_package_admin;
@@ -47,11 +44,6 @@ public class Messages extends NLS {
 	public static String ProcessRepo_location_not_url;
 	public static String ProcessRepo_must_be_local;
 
-	public static String Repo2Runnable_no_destination_repository;
-	public static String Repo2Runnable_no_engine;
-
-	public static String Repo2RunnableTask_error_transforming_repository;
-
 	public static String SlicingOption_invalid_platform;
 	public static String exception_invalidDestination;
 	public static String exception_invalidSource;
diff --git a/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/messages.properties b/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/messages.properties
index 0e0a816..14cc864 100644
--- a/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.repository.tools/src/org/eclipse/equinox/p2/internal/repository/tools/messages.properties
@@ -11,15 +11,10 @@
 AbstractApplication_no_valid_destinations=Unable to locate a valid destination repository.
 AbstractRepositoryTask_unableToFind=Unable to find: {0} 
 
-CompositeRepository_default_metadataRepo_name=Composite Metadata Repository
-CompositeRepository_default_artifactRepo_name=Composite Artifact Repository
-CompositeRepository_create_colocated_composite_repos=Creating co-located composite repositories.
 CompositeRepository_composite_repository_exists=Composite repository already exists at location: {0}
-CompositeRepository_creating_composite_repositories=Creating Composite Repositories
 CompositeRepository_default_artifactRepo_name=Composite Artifact Repository
 CompositeRepository_default_metadataRepo_name=Composite Artifact Repository
 
-no_valid_IUs=Need to specify either a non-empty source metadata repository or a valid list of IUs.
 no_artifactRepo_manager=Unable to acquire artifact repository manager service.
 no_metadataRepo_manager=Unable to acquire metadata repository manager service.
 no_package_admin=Unable to acquire package admin service.
diff --git a/bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/Messages.java b/bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/Messages.java
index 0fb7714..64ad6a2 100644
--- a/bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/Messages.java
@@ -65,9 +65,7 @@ public class Messages extends NLS {
 	public static String TransportErrorTranslator_UnknownErrorCode;
 	public static String TransportErrorTranslator_UnknownHost;
 
-	public static String fetching_0_1_at_2;
 	public static String fetching_0_from_1_2_at_3;
-	public static String fetching_0_1_of_2_at_3;
 	public static String fetching_0_from_1_2_of_3_at_4;
 	public static String connection_to_0_failed_on_1_retry_attempt_2;
 
diff --git a/bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/messages.properties b/bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/messages.properties
index 45d2af7..b391181 100644
--- a/bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/messages.properties
@@ -64,9 +64,7 @@ TransportErrorTranslator_MalformedRemoteFileReference=Malformed reference to rem
 TransportErrorTranslator_UnableToConnectToRepository_0=Unable to connect to repository {0}
 TransportErrorTranslator_UnknownErrorCode=HTTP Server Unknown HTTP Response Code ({0}):{1}
 TransportErrorTranslator_UnknownHost=Unknown Host: {0}
-fetching_0_1_at_2=Fetching {0} ({1} at {2}/s)
 fetching_0_from_1_2_at_3=Fetching {0} ({2} at {3}/s) from {1}
-fetching_0_1_of_2_at_3=Fetching {0} ({1} of {2} at {3}/s)
 fetching_0_from_1_2_of_3_at_4=Fetching {0} ({2} of {3} at {4}/s) from {1}
 FileTransport_reader=File Transport Reader
 connection_to_0_failed_on_1_retry_attempt_2=Connection to {0} failed on {1}. Retry attempt {2} started
diff --git a/bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/Messages.java b/bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/Messages.java
index 57f26fc..0d73c2a 100644
--- a/bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/Messages.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2007 IBM Corporation and others.
+ * Copyright (c) 2007, 2009 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
@@ -21,22 +21,18 @@ public class Messages extends NLS {
 	}
 
 	public static String BackupStore_0_of_1_items_restored;
-	public static String BackupStore_backup_root_for_0_not_determinable;
 	public static String BackupStore_backupCopy_closed_store;
 	public static String BackupStore_can_not_close_tcp_port;
 	public static String BackupStore_can_not_copy_directory;
-	public static String BackupStore_can_not_create_backup_dir;
 	public static String BackupStore_can_not_create_dummy;
 	public static String BackupStore_can_not_delete_after_copy_0;
 	public static String BackupStore_can_not_delete_tmp_file;
-	public static String BackupStore_can_not_move_file_to;
 	public static String BackupStore_can_not_remove;
 	public static String BackupStore_can_not_remove_bu_directory;
 	public static String BackupStore_directory_file_mismatch;
 	public static String BackupStore_directory_not_empty;
 	public static String BackupStore_errors_while_restoring_see_log;
 	public static String BackupStore_externally_modified_0_of_1_restored;
-	public static String BackupStore_file_0_must_be_related_to_1;
 	public static String BackupStore_file_directory_mismatch;
 	public static String BackupStore_file_not_found;
 	public static String BackupStore_manual_restore_needed;
@@ -50,7 +46,6 @@ public class Messages extends NLS {
 	public static String could_not_obtain_download_cache;
 	public static String download_cache_not_writeable;
 	public static String unzipping;
-	public static String restoring;
 	public static String param_not_set;
 	public static String copy_failed;
 	public static String failed_backup_restore;
diff --git a/bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/messages.properties b/bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/messages.properties
index 7d22d27..1dbe1a1 100644
--- a/bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.touchpoint.natives/src/org/eclipse/equinox/internal/p2/touchpoint/natives/messages.properties
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2007, 2008 IBM Corporation and others.
+# Copyright (c) 2007, 2009 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License v1.0
 # which accompanies this distribution, and is available at
@@ -9,23 +9,19 @@
 #     IBM Corporation - initial API and implementation
 ###############################################################################
 
-BackupStore_backup_root_for_0_not_determinable=Can not determine backup root for file: {0}
 BackupStore_backupCopy_closed_store=Can not perform backup on closed backup store.
 BackupStore_0_of_1_items_restored={0} items out of {1} items restored.
 BackupStore_can_not_close_tcp_port=Could not close tcp socket for port: {0}
 BackupStore_can_not_copy_directory=Can not copy a directory: {0}
-BackupStore_can_not_create_backup_dir=Can not create backup directory: {0}
 BackupStore_can_not_create_dummy=Can not create dummy file: {0}
 BackupStore_can_not_delete_after_copy_0=File that was copied to backup could not be deleted: {0}
 BackupStore_can_not_delete_tmp_file=Can not delete temporary file - it is safe to delete it manually: {0}
-BackupStore_can_not_move_file_to=Could not move the file: {0} to: {1}
 BackupStore_can_not_remove=Can not remove : {0}
 BackupStore_can_not_remove_bu_directory=Could not remove temporary backup directory (it is safe to manually delete it and its contents): {0}
 BackupStore_directory_file_mismatch=File already backed up as a directory: {0}
 BackupStore_directory_not_empty=Directory is not empty: {0}
 BackupStore_errors_while_restoring_see_log=Errors while restoring - see earlier logged errors
 BackupStore_externally_modified_0_of_1_restored=Backup store modified externally\! {0} items out of {1} items restored. Remaining items can not be found.
-BackupStore_file_0_must_be_related_to_1=File: {0} must be related to File: {1}.
 BackupStore_file_directory_mismatch=File already backed up as a file - and is now required as a directory: {0}
 BackupStore_file_not_found=File does not exist: {0}.
 BackupStore_manual_restore_needed=Manual restore of backup needed for: {0}
@@ -35,7 +31,6 @@ BackupStore_restore_closed_store=Can not perform restore on closed backup store
 
 action_0_failed_file_1_doesNotExist=The action {0} failed - file {1} does not exist
 artifact_not_available=The artifact for {0} is not available.
-restoring=Restoring {0}
 unzipping=Extracting {0}
 param_not_set=The \"{0}\" parameter was not set in the \"{1}\" action.
 artifact_repo_not_found=The artifact repository manager could not be found.
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java b/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java
index 27c159f..af52673 100644
--- a/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java
+++ b/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java
@@ -33,8 +33,6 @@ public class ProvAdminUIMessages extends NLS {
 	public static String MetadataRepositoriesView_AddRepositoryLabel;
 	public static String MetadataRepositoriesView_ChooseProfileDialogTitle;
 	public static String MetadataRepositoriesView_RemoveRepositoryTooltip;
-	public static String MetadataRepositoriesView_RevertLabel;
-	public static String MetadataRepositoriesView_UnexpectedRevertError;
 	public static String ArtifactRepositoriesView_AddRepositoryTooltip;
 	public static String ArtifactRepositoriesView_AddRepositoryLabel;
 	public static String ArtifactRepositoriesView_RemoveRepositoryTooltip;
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties b/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties
index ee6d78a..cad60fe 100644
--- a/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties
@@ -25,9 +25,7 @@ MetadataRepositoriesView_AddRepositoryLabel=Add repository...
 MetadataRepositoriesView_AddRepositoryTooltip=Add a new metadata repository
 RepositoriesView_RemoveCommandLabel=Remove
 MetadataRepositoriesView_RemoveRepositoryTooltip=Remove the selected metadata repositories
-MetadataRepositoriesView_RevertLabel=Revert to Snapshot
 MetadataRepositoriesView_ChooseProfileDialogTitle=Choose Profile
-MetadataRepositoriesView_UnexpectedRevertError=Unexpected error while reverting profile.
 ArtifactRepositoriesView_RemoveRepositoryOperationLabel=Remove Artifact Repositories
 ArtifactRepositoriesView_AddRepositoryLabel=Add repository...
 ArtifactRepositoriesView_AddRepositoryTooltip=Add a new artifact repository
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
index f2f7138..cab15b6 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
@@ -24,7 +24,6 @@ public class ProvSDKMessages extends NLS {
 		// load message values from bundle file
 		NLS.initializeMessages(BUNDLE_NAME, ProvSDKMessages.class);
 	}
-	public static String Error_Saving_Preferences;
 	public static String Handler_CannotLaunchUI;
 	public static String Handler_SDKUpdateUIMessageTitle;
 	public static String InstallNewSoftwareHandler_LoadRepositoryJobLabel;
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
index eeccf56..332c327 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
@@ -10,7 +10,6 @@
 ###############################################################################
 
 
-Error_Saving_Preferences=Error saving update preferences
 Handler_CannotLaunchUI=Cannot complete the request.  This installation has not been configured properly for Software Updates.
 Handler_SDKUpdateUIMessageTitle=Software Updates
 InstallNewSoftwareHandler_LoadRepositoryJobLabel=Contacting Software Sites
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
index d0509a6..f547048 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
@@ -37,7 +37,6 @@ public class ProvUIMessages extends NLS {
 	public static String AcceptLicensesWizardPage_Title;
 	public static String ApplicationInRestartDialog;
 	public static String ApplyProfileChangesDialog_ApplyChanges;
-	public static String CategoryElementCollector_Uncategorized;
 	public static String ColocatedRepositoryManipulator_AddSiteOperationLabel;
 	public static String ColocatedRepositoryManipulator_GotoPrefs;
 	public static String ColocatedRepositoryManipulator_ManageSites;
@@ -68,7 +67,6 @@ public class ProvUIMessages extends NLS {
 	public static String ProfileModificationAction_NoExplanationProvided;
 	public static String ProfileModificationAction_ResolutionOperationLabel;
 	public static String ProfileModificationWizardPage_DetailsLabel;
-	public static String ProfileModificationWizardPage_NothingSelected;
 	public static String ProfileModificationWizardPage_ResolutionOperationLabel;
 	public static String ProfileModificationWizardPage_UnexpectedError;
 	public static String ProfileSnapshots_Label;
@@ -87,9 +85,6 @@ public class ProvUIMessages extends NLS {
 	public static String ProvDropAdapter_UnsupportedDropOperation;
 
 	// Provisioning operations
-	public static String ProvisioningOperation_ExecuteErrorTitle;
-	public static String ProvisioningOperation_RedoErrorTitle;
-	public static String ProvisioningOperation_UndoErrorTitle;
 	public static String ProvisioningOperationRunner_ErrorExecutingOperation;
 	public static String InstalledSoftwarePage_NoProfile;
 	public static String InstallIUOperationLabel;
@@ -144,7 +139,6 @@ public class ProvUIMessages extends NLS {
 	public static String RepositoryManipulationPage_Edit;
 	public static String RepositoryManipulationPage_RefreshConnection;
 	public static String RepositoryManipulationPage_RefreshOperationCanceled;
-	public static String RepositoryManipulationPage_RefreshOperationLabel;
 	public static String RepositoryManipulationPage_Remove;
 	public static String RepositoryManipulationPage_RemoveConfirmMessage;
 	public static String RepositoryManipulationPage_RemoveConfirmSingleMessage;
@@ -162,9 +156,6 @@ public class ProvUIMessages extends NLS {
 	public static String ResolutionWizardPage_NoSelections;
 	public static String ResolutionWizardPage_WarningInfoStatus;
 
-	public static String AddColocatedRepositoryAction_Label;
-	public static String AddColocatedRepositoryAction_Tooltip;
-	public static String AddColocatedRepositoryDialog_AddSiteTitle;
 	// Dialogs
 	public static String AddRepositoryDialog_DuplicateURL;
 	public static String AddRepositoryDialog_InvalidURL;
@@ -205,7 +196,6 @@ public class ProvUIMessages extends NLS {
 	public static String UpdateAction_UpdatesAvailableMessage;
 	public static String UpdateAction_UpdatesAvailableTitle;
 	public static String PlanAnalyzer_IgnoringInstall;
-	public static String PlanAnalyzer_IgnoringUninstall;
 	public static String PlanAnalyzer_LockedImpliedUpdate0;
 	public static String PlanAnalyzer_PartialInstall;
 	public static String PlanAnalyzer_PartialUninstall;
@@ -226,7 +216,6 @@ public class ProvUIMessages extends NLS {
 	public static String PlanStatusHelper_PromptForUpdateManagerUI;
 	public static String PlatformUpdateTitle;
 	public static String PlatformRestartMessage;
-	public static String Policy_MultiplePolicyRegistrationsWarning;
 	public static String ProvUI_ErrorDuringApplyConfig;
 	public static String ProvUI_InformationTitle;
 	public static String ProvUI_InstallDialogError;
@@ -246,10 +235,8 @@ public class ProvUIMessages extends NLS {
 	public static String QueriedElementWrapper_NoItemsExplanation;
 	public static String QueriedElementWrapper_SiteNotFound;
 	public static String ColocatedRepositoryManipulator_SiteNotFoundDescription;
-	public static String QueryableArtifactRepositoryManager_RepositoryQueryProgress;
 	public static String QueryableMetadataRepositoryManager_LoadRepositoryProgress;
 	public static String QueryableMetadataRepositoryManager_MultipleRepositoriesNotFound;
-	public static String QueryableMetadataRepositoryManager_RepositoryQueryProgress;
 	public static String QueryableProfileRegistry_QueryProfileProgress;
 	public static String QueryableUpdates_UpdateListProgress;
 	public static String SizeComputingWizardPage_SizeJobTitle;
@@ -257,12 +244,8 @@ public class ProvUIMessages extends NLS {
 	public static String RevertDialog_ConfigContentsLabel;
 	public static String RevertDialog_ConfigsLabel;
 	public static String RevertDialog_ConfirmRestartMessage;
-	public static String RevertDialog_Description;
-	public static String RevertDialog_PageTitle;
-	public static String RevertDialog_RevertError;
 	public static String RevertDialog_RevertOperationLabel;
 	public static String RevertDialog_Title;
-	public static String RevertProfileWizardPage_ErrorRetrievingHistory;
 	public static String RollbackProfileElement_CurrentInstallation;
 	public static String RollbackProfileElement_InvalidSnapshot;
 	public static String SelectableIUsPage_Select_All;
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
index a3683e0..2e9e41b 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
@@ -16,7 +16,6 @@ ProfileModificationAction_NoChangeRequestProvided=Could not interpret the reques
 ProfileModificationAction_NoExplanationProvided=Unexpected error while processing the request.
 ProfileModificationAction_ResolutionOperationLabel=Calculating Requirements
 ProfileModificationWizardPage_DetailsLabel=Details
-ProfileModificationWizardPage_NothingSelected=You must check some items in order to continue.
 ProfileModificationWizardPage_ResolutionOperationLabel=Calculating Requirements
 ProfileModificationWizardPage_UnexpectedError=An unexpected error occurred.  This wizard is no longer valid with the current selections.
 ProfileSnapshots_Label=Installation History
@@ -28,9 +27,6 @@ ProvisioningUtil_NoPlannerFound=No provisioning planner was found
 ProvisioningUtil_NoDirectorFound=No provisioning director was found
 ProvisioningUtil_NoEngineFound=No provisioning engine was found
 
-ProvisioningOperation_ExecuteErrorTitle=Error while executing the "{0}" operation
-ProvisioningOperation_RedoErrorTitle=Error while redoing the "{0}" operation
-ProvisioningOperation_UndoErrorTitle=Error while undoing the "{0}" operation
 ProvisioningOperationRunner_ErrorExecutingOperation=Unexpected error while trying to run {0}
 InstalledSoftwarePage_NoProfile=This installation has not been configured for showing the installed software.  See the error log for details.
 InstallIUOperationLabel=Install
@@ -102,7 +98,6 @@ RepositoryManipulationPage_NameColumnTitle=Name
 RepositoryManipulationPage_RefreshOperationCanceled=Operation canceled.
 RepositoryManipulationPage_Edit=&Edit
 RepositoryManipulationPage_RefreshConnection=Test &Connection
-RepositoryManipulationPage_RefreshOperationLabel=Refresh
 RepositoryManipulationPage_Remove=&Remove
 RepositoryManipulationPage_RemoveConfirmMessage=Are you sure you want to remove the selected items?
 RepositoryManipulationPage_RemoveConfirmSingleMessage=Are you sure you want to remove {0}?
@@ -129,9 +124,6 @@ AcceptLicensesWizardPage_RejectSingle=I &do not accept the terms of the license
 AcceptLicensesWizardPage_ReviewExtraLicensesDescription=Licenses must be reviewed before the software can be installed.  This includes licenses for software required to complete the install.
 AcceptLicensesWizardPage_ReviewLicensesDescription=Licenses must be reviewed and accepted before the software can be installed.
 AcceptLicensesWizardPage_Title=Review Licenses
-AddColocatedRepositoryAction_Label=&Add Site...
-AddColocatedRepositoryAction_Tooltip=Add a site used to access the available software
-AddColocatedRepositoryDialog_AddSiteTitle=Add Site
 AddRepositoryDialog_Title=Add Repository
 AddRepositoryDialog_DuplicateURL=Duplicate location
 AddRepositoryDialog_InvalidURL=The provided location is not valid
@@ -155,7 +147,6 @@ UpdateWizardPage_Title=Update Details
 UserValidationDialog_PasswordLabel=Password:
 UserValidationDialog_SavePasswordButton=Save password
 UserValidationDialog_UsernameLabel=Username:
-CategoryElementCollector_Uncategorized=Uncategorized
 ColocatedRepositoryManipulator_AddSiteOperationLabel=Add Site
 ColocatedRepositoryManipulator_GotoPrefs=Go to the <a>Available Software Sites</a> preferences
 ColocatedRepositoryManipulator_ManageSites=&Manage Sites...
@@ -169,7 +160,6 @@ PreselectedIUInstallWizard_Title=Install
 PreselectedIUInstallWizard_Description=Check the items that you wish to install.
 
 PlanAnalyzer_IgnoringInstall="{0}" is not applicable to the current configuration and will not be installed.
-PlanAnalyzer_IgnoringUninstall="{0}" cannot be uninstalled because other installed software depends on it.
 PlanAnalyzer_LockedImpliedUpdate0={0} will be ignored because it is already installed, and updates are not permitted.
 PlanAnalyzer_PartialInstall="{0}" is partially installed because other installed software requires it.  The remainder will be installed.
 PlanAnalyzer_PartialUninstall= "{0}" cannot be fully uninstalled because other installed software requires it.  The parts that are not required will be uninstalled.
@@ -246,10 +236,8 @@ QueriedElementWrapper_NoCategorizedItemsExplanation=There are no categorized ite
 QueriedElementWrapper_NoItemsExplanation=There are no items available
 QueriedElementWrapper_SiteNotFound=Could not find {0}
 ColocatedRepositoryManipulator_SiteNotFoundDescription=The site's location may be entered incorrectly, or the site might be offline.  You can correct the location in the 'Available Software Sites' preferences by removing the site and adding it with the proper location.  If the site name appears to be correct, you can test the connection in the 'Available Software Sites' preferences.  The site may be unreachable, or you may have a problem with your network connection.
-QueryableArtifactRepositoryManager_RepositoryQueryProgress=Getting artifact repositories
 QueryableMetadataRepositoryManager_LoadRepositoryProgress=Contacting {0}
 QueryableMetadataRepositoryManager_MultipleRepositoriesNotFound=Some sites could not be found.  See the error log for more detail.
-QueryableMetadataRepositoryManager_RepositoryQueryProgress=Retrieving information from {0}
 QueryableProfileRegistry_QueryProfileProgress=Getting profiles
 QueryableUpdates_UpdateListProgress=Assembling list of updates
 SelectableIUsPage_Select_All=Select &All
@@ -260,13 +248,9 @@ SizingPhaseSet_PhaseSetName=Compute sizes
 RevertDialog_ConfigContentsLabel=&Configuration contents:
 RevertDialog_ConfigsLabel=&Previous configurations:
 RevertDialog_ConfirmRestartMessage=This operation requires restarting the workbench.  Would you like to proceed?
-RevertDialog_Description=Choose a previously installed configuration to which you would like to revert.
-RevertDialog_PageTitle=Revert to a Previous Configuration
-RevertDialog_RevertError=Unexpected error while reverting configuration.  The configuration was not reverted.
 RevertDialog_RevertOperationLabel=Revert configuration
 RevertDialog_Title=Revert Software Configuration
 
-RevertProfileWizardPage_ErrorRetrievingHistory=Unexpected error while getting the history list.
 RollbackProfileElement_CurrentInstallation=Current Installation
 RollbackProfileElement_InvalidSnapshot=The configuration snapshot is no longer valid.
 TrustCertificateDialog_Details=Details
@@ -275,5 +259,3 @@ TrustCertificateDialog_Title=Do you trust these certificates?
 PlanStatusHelper_PromptForUpdateManagerUI=A feature that you have selected uses install procedures that are not compatible with the current installation support.  This feature can only be installed by the older Update Manager.  \
 \
 Do you want to launch the older Update Manager?
-
-Policy_MultiplePolicyRegistrationsWarning=Multiple registrations for the p2 UI Policy were found.  Using first found.
diff --git a/bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/Messages.java b/bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/Messages.java
index fd08658..8b54170 100644
--- a/bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/Messages.java
+++ b/bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/Messages.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2008 IBM Corporation and others.
+ * Copyright (c) 2008, 2009 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
@@ -22,15 +22,8 @@ public class Messages extends NLS {
 	public static String ErrorReadingDigest;
 	public static String ErrorReadingFeature;
 	public static String ErrorReadingSite;
-	public static String InvalidRepositoryLocation;
 	public static String Error_Generation;
 
-	public static String io_failedRead;
-	public static String ecf_configuration_error;
-	public static String repo_loading;
-
-	public static String repoMan_internalError;
-
 	public static String DefaultFeatureParser_IdOrVersionInvalid;
 	public static String DefaultSiteParser_NoSiteTag;
 	public static String DefaultSiteParser_WrongParsingStack;
diff --git a/bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/messages.properties b/bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/messages.properties
index b993602..de4677d 100644
--- a/bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.updatesite/src/org/eclipse/equinox/internal/p2/updatesite/messages.properties
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2008 IBM Corporation and others.
+# Copyright (c) 2008, 2009 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License v1.0
 # which accompanies this distribution, and is available at
@@ -11,12 +11,7 @@
 ErrorReadingDigest=Error reading site digest {0}.
 ErrorReadingFeature=Error reading feature {0}.
 ErrorReadingSite=Error reading update site {0}.
-InvalidRepositoryLocation=Invalid repository location {0}.
-io_failedRead=Unable to read repository at {0}.
-repoMan_internalError=Internal error.
-repo_loading = Loading the repository {0}
 Error_Generation=Problems generating update site metadata {0}.
-ecf_configuration_error=Transport initialization error.
 DefaultFeatureParser_IdOrVersionInvalid= Error parsing feature stream. The unique identifier or the version is null or empty for the State: \"{2}\": unique identifier=\"{0}\" version=\"{1}\".
 DefaultSiteParser_NoSiteTag= Error parsing site stream. Unable to find root element \"site\" in the stream.
 DefaultSiteParser_WrongParsingStack= Internal Error parsing site stream. Unexpected Parsing Stack: \"{0}\"
